linux/network_device_ops/check_pvrdma_support_status.yml

```diff
@@ -9,7 +9,7 @@
     skip_msg: "{{ guest_os_ansible_distribution }} doesn't support PVRDMA"
     skip_reason: "Not Supported"
   when: >
-    ('Flatcar' in guest_os_ansible_distribution or
+    ('Flatcar' == guest_os_ansible_distribution or
      guest_os_ansible_distribution in ['Fedora', 'VMware Photon OS'])
```

Comment:
This condition can be replaced with `guest_os_ansible_distribution in ['Fedora', 'VMware Photon OS', 'Flatcar']`